### DISCUSSION TITLE
Is there a way to set the directory with an install script?

### DISCUSSION THREAD

**Discussion by denizonm** [lutris/lutris#5479] [Q&A]
Hello,
I'm not talking about the working directory, i mean the one on the bottom of the "About This Game" section.
The one that lutris asks if you want deleted when you are removing the game or moves when you want to move the game.

Thank you!

**Comment by strycore** [comment]
I don't fully understand your question I'm afraid....

The point of install scripts is to let users chose where they want to install their game. Lutris determines if a game needs an install directory and if so the user will pick one. It is not up to the script maintainer to decide where the game goes.

> one on the bottom of the "About This Game" section

I am not aware of a "About This Game" section. Afaik, we don't have one.

> The one that lutris asks if you want deleted 

Yes that is the game directory selected by the user. 


Eventually, we'll have ways to automate game installs more. So you'll be able to automate where games are installed. But this is stil on the user's end and it is not up to the script to decide where a game goes (thankfully, that would be very creepy and insecure)

**Comment by denizonm** [comment]
Thank you for your fast response.

My bad, I was translating it on the fly since I couldn't find a way to change the language without changing the overall language of the whole system.

I've looked up on videos and it's the "Game Info" part in the game config menu.

I should have clarified what I meant, I am writing local install scripts for preexisting linux games.
Is there a way to set the directory like that?

This is what i have right now for reference.

```
name: game name
game_slug: slug
version: version
slug: slug
runner: linux

script:
  game:
    exe: "directory/to/game/game.sh"
```

Thanks again.

**Comment by strycore** [comment]
Ok, I understand better now. We have a similar feature to install games previously installed with lutris but I don't think this is what you want. What you want is this: https://github.com/lutris/lutris/pull/3029 but it needs a bunch of work to be compatible with our current codebase.
